modules/gatsby/src/components/github/github-contributors.jsx

```diff
@@ -0,0 +1,43 @@
+import React, {Component} from 'react';
+
+export default class GithubContributors extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      response: window.fetchContributorsResponse
+    };
+  }
+
+  componentDidMount() {
+    if (window.fetchContributorsResponse) return;
+
+    const {project} = this.props;
+    fetch(`https://api.github.com/repos/${project}/contributors`)
+      .then(response => response.json())
+      .then(response => {
+        window.fetchContributorsResponse = response;
+        this.setState({response});
+      });
+  }
+
+  render() {
+    const {response} = this.state;
+    const contributors = response || [];
+    return contributors.map(contributor =>
+      contributor ? (
+        <a
+          target="_blank"
+          rel="noopener noreferrer"
+          href={contributor.html_url}
+          className="Contributor"
+          key={contributor.id}
+        >
+          <img src={contributor.avatar_url} width="100%" alt="" />
+          <span>{contributor.login}</span>
+        </a>
+      ) : (
+        <div className="Contributor" key={contributor.id} />
```

Comment:
How does this work? `contributor` is "falsy" in this branch. Won't `contributor.id` crash?

Reply:
Copy-pasted this.